docs/index.md

```diff
@@ -1,215 +1,53 @@
 ---
 hide:
   - toc
+  - navigation
+  - footer
 ---
 
-# :ibis-logo: Ibis
+# <span style="font-size: 1.5em; margin: 0">:ibis-logo: The Ibis Project</span>
 
-## Expressive analytics in Python at any scale.
+## Data analytics at scale
 
-<script 
-    src="https://asciinema.org/a/yp5Ww4XKyjJsUCXkEz5or9rPq.js"
-    data-autoplay="true"
-    data-preload="true"
-    data-loop="true"
-    data-i="4"
-    data-rows="20"
-    id="asciicast-yp5Ww4XKyjJsUCXkEz5or9rPq"
-    async>
-</script>
+The flexibility of Python with the scale and performance of modern SQL.
 
-## Installation
+[Install](/install){ .md-button .md-button--primary }
+[Tutorial](/tutorial){ .md-button }
 
```

Comment:
I like the style of the buttons but they look a bit weird underneath one half line of text and aligned to the left.  There's a lot of empty space to the right--is it possible to center and dynamically pad them or something?

Reply:
Yeah, I agree.  I can't figure out how to do it reasonably easily right now, however.  I think we just need to wrap the two elements in an enclosing `<div style="margin: auto;">`.

Reply:
FTFY

![image](https://user-images.githubusercontent.com/417981/200833853-69e1dc1e-22ea-4fae-9374-70318bb05234.png)